echo/server/prod-worker-cpu.sh

```
@@ -1,2 +1,18 @@
+
+PROCESSES=${CPU_WORKER_PROCESSES:-2}
+THREADS=${CPU_WORKER_THREADS:-4}
+
+echo "Configuration:"
+echo "  Processes: $PROCESSES | Threads: $THREADS"
+echo "  Capacity per pod: $((PROCESSES * THREADS)) concurrent tasks"
+echo "📊 Scale with K8s replicas"
+
+exec dramatiq \
+  --queues cpu \
+  --processes $PROCESSES \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)